### PR TITLE
New extension - Postman

### PR DESCRIPTION
**Pull request by tarikyildizci**
## Description

Postman extension has one command: Send Request. It lists all your collections, and requests inside. If you select a GET request, you can fill out params and environment variables and send the request. Then you can see and copy the response.

## Screencast

postman-1
postman-2
postman-3
postman-4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/postman/src/sendRequest.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      {data ? (
+        (data as CollectionsResponseType).collections.map((collection) => (
+          <List.Item
+            key={collection.id}
+            title={collection.name}
+            icon={{ source: Icon.Folder, tintColor: Color.Orange }}
+            actions={
+              <ActionPanel>
+                <Action
+                  title="Open Collection"
```
Comment (pernielsentikaer) [inline extensions/postman/src/sendRequest.tsx:42]: Missing an icon

File: extensions/postman/src/components/RequestBuilder.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+}) => {
+  const { push } = useNavigation()
+
+  return (
+    <Form
+      navigationTitle={name + " Form"}
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm
+            title="Send Request"
```
Comment (pernielsentikaer) [inline extensions/postman/src/components/RequestBuilder.tsx:35]: Maybe also an icon here

File: extensions/postman/package.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  "categories": [
+    "Applications",
+    "Developer Tools",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "sendRequest",
+      "title": "Send Request",
```
Comment (pernielsentikaer) [inline extensions/postman/package.json:17]:
```suggestion
      "title": "Send Request",
      "subtitle": "Postman",
```

File: extensions/postman/src/components/CollectionListItem.tsx [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+const Actions: React.FC<{
+  urlInfo: ReturnType<typeof parseRequest>
+  requestDetails: RequestDetailsType
+}> = ({ urlInfo, requestDetails }) => {
+  if (requestDetails.request.method !== "GET") {
+    return (
+      <ActionPanel>
+        <Action
+          title="Send Request"
```
Comment (pernielsentikaer) [inline extensions/postman/src/components/CollectionListItem.tsx:55]: missing icon here

File: extensions/postman/src/components/CollectionListItem.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+            })
+          }
+        />
+      </ActionPanel>
+    )
+  }
+  if (urlInfo) {
+    return (
+      <ActionPanel>
+        <Action.Push
```
Comment (pernielsentikaer) [inline extensions/postman/src/components/CollectionListItem.tsx:71]: missing icon here

File: extensions/postman/src/components/CollectionListItem.tsx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+          }
+          title="Send Request"
+        />
+      </ActionPanel>
+    )
+  }
+  return (
+    <ActionPanel>
+      <Action
+        title="Send Request"
```
Comment (pernielsentikaer) [inline extensions/postman/src/components/CollectionListItem.tsx:98]: missing icon here

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by tarikyildizci** [comment]
@pernielsentikaer Thanks for the quick review! I added all missing icons, and the subtitle in the package.json.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋

Tested with success, let's go! 🧑‍🚒

**Pull request merged**